konan/konan.properties

```diff
@@ -25,6 +25,7 @@ dependencyProfiles = default alt
 airplaneMode = false
 # if true, ignores Xcode version check.
 ignoreXcodeVersionCheck = false
+useFixedXcodeVersion = 10.2
```

Comment:
Doesn't this force a user to use Xcode 10.2?

Reply:
Maybe filter that out in release bundle.

Reply:
Not sure if I like approach taken here, let's discuss.